podvm/Dockerfile.podvm

```diff
@@ -0,0 +1,22 @@
+# Copyright Confidential Containers Contributors
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+# Builds pod vm image inside container
+#
+ARG BUILDER_IMG="quay.io/confidential-containers/cloud-api-adaptor-podvm-builder"
+
+FROM ${BUILDER_IMG} AS podvm_builder
+
+ARG CLOUD_PROVIDER=libvirt
```

Comment:
Should we introduce a `generic` or `null` provider for this image creation rather than use `libvirt`?

Reply:
We don't have any generic or null provider and that needs to be added first.  Let's keep `libvirt` as default now and we can change it once null or generic provider code is added. 
Does it sound reasonable ?